pandas/core/arrays/period.py

```diff
@@ -457,7 +457,7 @@ def value_counts(self, dropna=False):
 
     # --------------------------------------------------------------------
 
-    def shift(self, periods=1):
+    def shift(self, periods=1, fill_value=None):
```

Comment:
add to docstring. Note that the default is `NaT`

Reply:
added

Reply:
added